### PR TITLE
Add kubecon-2023 blog

### PR DESCRIPTION
**Pull request by LFSb**

### CODE REVIEW

File: content/blog/kubecon-2023.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Kubecon 2023 is a wrap! In fact, it's already a couple of weeks in the past. As only my second Kubecon ever (Kubecon Valencia 2022 was my first), it leaves me with a lot of mixed feelings. 
+
+Yes, it was a well run event, it was filled with interesting talks and like last year, I learned about a lot of stuff happening outside the bubble of my current assignment, which is one of the main attractions of being a consultant at Skyworkz, and going to a tech conference to meet people.
+
+However, KubeCon also left me thinking that I am no longer its target audience. Why? 
+
+We'll get to that, let's first lead off with couple of stand-out talks that I feel are worth a (re)-watch.
+
+## 1.  Enabling Real-Time Media in Kubernetes, Giles Heron
+https://youtu.be/1_cxXzhY4Xg
```
Comment (bennycornelissen) [inline content/blog/kubecon-2023.md]: This should be a link, or you can try something like this below, that _should_ potentially embed the video in the blog, but I'm not sure if we have that enabled for the Skyworkz blog or not 😉  

```markdown
{{< youtube 1_cxXzhY4Xg >}}
```

File: content/blog/kubecon-2023.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+id: blog
+title: Kubecon, is it still for me?
+description: Lee looks back on his second Kubecon, lists some standout-talks and discusses his takeaway.
+summary: Lee looks back on his second Kubecon, lists some standout-talks and discusses his takeaway.
```
Comment (bennycornelissen) [inline content/blog/kubecon-2023.md:5]: This part is also used for the blog index page, so you may want to make this slightly longer.

File: content/blog/kubecon-2023.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+---
+Kubecon 2023 is a wrap! In fact, it's already a couple of weeks in the past. As only my second Kubecon ever (Kubecon Valencia 2022 was my first), it leaves me with a lot of mixed feelings. 
+
+Yes, it was a well run event, it was filled with interesting talks and like last year, I learned about a lot of stuff happening outside the bubble of my current assignment, which is one of the main attractions of being a consultant at Skyworkz, and going to a tech conference to meet people.
+
+However, KubeCon also left me thinking that I am no longer its target audience. Why? 
+
+We'll get to that, let's first lead off with couple of stand-out talks that I feel are worth a (re)-watch.
+
+## 1.  Enabling Real-Time Media in Kubernetes, Giles Heron
```
Comment (bennycornelissen) [inline content/blog/kubecon-2023.md]: From a formatting perspective I'm not entirely convinced heading2 works well here, especially as they get longer.

File: content/blog/kubecon-2023.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+slug: kubecon-2023
+date: 2023-05-05
+---
+Kubecon 2023 is a wrap! In fact, it's already a couple of weeks in the past. As only my second Kubecon ever (Kubecon Valencia 2022 was my first), it leaves me with a lot of mixed feelings. 
+
+Yes, it was a well run event, it was filled with interesting talks and like last year, I learned about a lot of stuff happening outside the bubble of my current assignment, which is one of the main attractions of being a consultant at Skyworkz, and going to a tech conference to meet people.
+
+However, KubeCon also left me thinking that I am no longer its target audience. Why? 
+
+We'll get to that, let's first lead off with couple of stand-out talks that I feel are worth a (re)-watch.
```
Comment (bennycornelissen) [inline content/blog/kubecon-2023.md]: I'd replace the colon with a period. It seems to work better that way for me.

**Comment by bennycornelissen** [comment]
Small nitpicks, otherwise LGTM. I'll be vacationing but if you need someone to sign off when you're happy with it, I can do that from my phone 😉

File: content/blog/kubecon-2023.md (hunk 4 above)
Comment (LFSb) [reply content/blog/kubecon-2023.md]: Agreed!

File: content/blog/kubecon-2023.md (hunk 1 above)
Comment (LFSb) [reply content/blog/kubecon-2023.md]: It works, but the embed is rather...large. It's more obvious than a clickable header though, so I'll go with that. Thanks!

File: content/blog/kubecon-2023.md (hunk 3 above)
Comment (LFSb) [reply content/blog/kubecon-2023.md]: Agreed, I'm going to say H3 looks a bit better.

File: content/blog/kubecon-2023.md (hunk 2 above)
Comment (LFSb) [reply content/blog/kubecon-2023.md:5]: Ok, I'll have a think about a summary with a little more substance then.

File: content/blog/kubecon-2023.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Kubecon mentioned this year that over half of their audience are first-time attendees. Last year when I was a first-timer, I recall a similar percentage being mentioned. With Kubecon's attendence breaking into the 10.000's, it is one of the largest open source conferences in Europe, so it definitely doing fine, but I'm very much in doubt if I'll be attending KubeCon Paris in 2024.
+
+The keynotes this year were largely uninteresting to me, and some misfortune (broken collarbone the week before) and underestimating (I tried to both participate in the Thursday CTF as well as attend every talk in the morning/afternoon sessions, didn't work out great) made for a generally unremarkable conference experience. It didn't help that most of my (ex)-colleagues attending were local, so nobody felt the need to head out into town and explore Amsterdam, a city most of us are already way too familiar with.
+
+So do I regret my time spent at KubeCon 2023? Definitely not. Will I be considering KubeCon Paris 2024? Probably. Will I be on the lookout for other conferences that align more with my interests? 
+
+Already looking. Suggestions welcome.
```
Comment (LFSb) [inline content/blog/kubecon-2023.md:54]: Comment to myself: It doesn't really make sense to ask for suggestions when there's no comment section under the blog...so write a better ending.

Comment (bennycornelissen) [reply content/blog/kubecon-2023.md:54]: You can link to your Twitter or something (if you use it)

**Review by bennycornelissen — APPROVED** [review]
LGTM

**Pull request merged**